pkgs/modules/bun/default.nix

```diff
@@ -39,18 +26,14 @@ in
     bun
   ];
 
-  replit.dev.languageServers.typescript-language-server.extensions = extensions;
+  replit.languageServers.typescript-language-server.extensions = extensions;
```

Comment:
Is this an unintentional change? I think we don't want the language server config in a Deployment.

Reply:
oh, it's unintentional yeah but i don't think we *ever* want the language server config in a deployment? why does lsp have to have `.dev`?

Reply:
Toby chose to make the options symmetrical between .dev and not .dev.